src/cmd/ksh93/sh/parse.c

```diff
@@ -1367,13 +1367,13 @@ static Shnode_t	*item(Lex_t *lexp,int flag)
 		return(t);
 	}
 	sh_lex(lexp);
+done:
 	/* redirection(s) following a compound command */
```

Comment:
Since this is now also used for redirections following a simple arithmetic command, the comment should be updated.